src/lib.rs

```diff
@@ -174,6 +168,127 @@ where
     rng().random()
 }
 
+/// Return an iterator over [`random()`] variates
+///
+/// This function is shorthand for
+/// <code>[rng()].[random_iter](Rng::random_iter)()</code>.
```

Comment:
Have you checked that `<code>` is handled as expected by rustdoc?

Such documentation looks a bit weird for my taste, I would write it roughly like this:
> This function is a shorthand for the `Rng::random_iter` method over `ThreadRng`.

But it's just my personal opinion and not a blocker.

Reply:
I've checked it locally